client/src/components/StreamField/blocks/BaseSequenceBlock.js

```diff
@@ -334,34 +359,7 @@ export class BaseSequenceChild extends EventEmitter {
   }
 
   collapse() {
-    this.contentElement.hide().attr('aria-hidden', 'true');
-    const label = this.getTextLabel({ maxLength: 50 });
-    this.titleElement.text(label || '');
-    this.collapsed = true;
-    this.contentElement
-      .get(0)
-      .dispatchEvent(
-        new CustomEvent('commentAnchorVisibilityChange', { bubbles: true }),
```

Comment:
This is now dispatched by the collapsible panels code, identically to what we’re doing here.